backend/controllers/GraphController.ts

```diff
@@ -4,30 +4,38 @@ import { HttpStatusCode } from 'axios';
 import { GraphService } from '../services/GraphService';
 import { HttpError } from '../types/HttpError';
 import { validateAuthToken } from '../services/TokenAuth';
+import logger from '../logger';
 
 export class GraphController extends BaseController {
   constructor(app: Express) {
     super(app, '/graph');
   }
 
   protected initializeRoutes(): void {
-    this.router.get('/:dsnpId/following', validateAuthToken, this.getFollowing.bind(this));
-    this.router.post('/:dsnpId/follow', validateAuthToken, this.postFollow.bind(this));
-    this.router.post('/:dsnpId/unfollow', validateAuthToken, this.postUnfollow.bind(this));
+    this.router.get('/:msaId/following', validateAuthToken, this.getFollowing.bind(this));
+    this.router.post('/:msaId/follow', validateAuthToken, this.postFollow.bind(this));
+    this.router.post('/:msaId/unfollow', validateAuthToken, this.postUnfollow.bind(this));
   }
 
+  /**
+   * Retrieves the list of users that a given user is following.
+   *
+   * @param req - The request object.
+   * @param res - The response object.
+   * @returns A Promise that resolves to the list of users that the given user is following.
+   */
   public async getFollowing(req: Request, res: Response) {
-    const msaId = req.params?.dsnpId;
+    const msaId = req.params?.msaId;
 
     if (!msaId) {
       return res.status(HttpStatusCode.NotFound).send();
     }
 
     try {
-      const follows = await GraphService.instance().then((service) => service.getPublicFollows(msaId));
+      const follows = await GraphService.getInstance().then((service) => service.getPublicFollows(msaId));
       return res.status(HttpStatusCode.Ok).send(follows);
     } catch (err) {
-      console.error('Error getting user follows', err);
+      logger.error(`Error getting user follows: ${err}`);
```

Comment:
Note, this is the canonical way to do error logging with the Pino logger we're using. Also, generally the format to use is `logger.<method>(objectToLog, logMessage)`. Note for error object, the property name of the error object must be `err` (configurable)
```suggestion
      logger.error({ err }, 'Error getting user follows');
```